client/src/index.html

```diff
@@ -16,6 +16,7 @@
         <script>
           window.customTicketList = [];
         </script>
+        <script src="https://apis.google.com/js/platform.js" async defer></script>
```

Comment:
@mredigonda why do we need this? can't we use an npm package for this?

Reply:
There is this NPM package: `https://www.npmjs.com/package/react-google-login`
Which I couldn't make it work.